code/modules/antagonists/traitor/objectives/steal.dm

```diff
@@ -0,0 +1,236 @@
+/datum/traitor_objective_category/steal_item
+	name = "Steal Item"
+	objectives = list(
+		list(
+			/datum/traitor_objective/steal_item/low_risk = 1,
+			/datum/traitor_objective/steal_item/low_risk_cap = 1,
+		) = 1,
+		/datum/traitor_objective/steal_item/somewhat_risky = 1,
+		/datum/traitor_objective/steal_item/risky = 1,
+		/datum/traitor_objective/steal_item/most_risky = 1
+	)
+
+/datum/traitor_objective/steal_item
+	name = "Steal \[ITEM] and place a bug on it. Hold it for \[TIME] minutes"
+	description = "Use the button below to materialize the bug within your hand, where you'll then be able to place it on the item. After that, you must keep it near you for \[TIME] minutes"
+
+	progression_minimum = 20 MINUTES
+	progression_reward = 5 MINUTES
+	telecrystal_reward = list(2, 4)
+
+	var/list/possible_items = list()
+	/// The current target item that we are stealing.
+	var/datum/objective_item/steal/target_item
+	/// A list of 2 elements, which contain the range that the time will be in. Represented in minutes.
+	var/hold_time_required = list(5, 15)
+	/// The current time fulfilled around the item
+	var/time_fulfilled = 0
+	/// The maximum distance between the bug and the objective taker for time to count as fulfilled
+	var/max_distance = 4
+	/// The bug that will be put onto the item
+	var/obj/item/traitor_bug/bug
+	/// Any special equipment that may be needed
+	var/list/special_equipment
+
+	abstract_type = /datum/traitor_objective/steal_item
+
+/datum/traitor_objective/steal_item/low_risk_cap
+	progression_minimum = 5 MINUTES
+	progression_maximum = 20 MINUTES
+
+	progression_reward = list(5 MINUTES, 10 MINUTES)
+	telecrystal_reward = 1
+	possible_items = list(
+		/datum/objective_item/steal/low_risk/techboard/borgupload,
+		/datum/objective_item/steal/low_risk/techboard/aiupload,
+		/datum/objective_item/steal/low_risk/aicard,
+	)
+
+/datum/traitor_objective/steal_item/low_risk
```

Comment:
I wouldn't call these low risk. They're unique items and require probably defeating their wielder to steal. In fact, if the chaplain has any off the arm replacement null rods, it also requires dismembering his arm in order to acquire.

I guess if anything this encourages purchasing the infiltrator suit, so I can't totally complain 😏 

That said, does this work with subtypes? Bone axes are fireaxe subtypes, so maybe it would be good to check if this works with those.